File: source/project/lib/models/cuenta_steam.dart
/// Modelo reconstruido a partir de los símbolos conservados en app.so.
class CuentaSteam {
  const CuentaSteam({
    required this.steamId,
    required this.nombre,
    required this.rutaCuenta,
    required this.rutaConfigCuenta,
    required this.rutaSteamCloudCuenta,
  });

  final String steamId;
  final String nombre;
  final String rutaCuenta;
  final String rutaConfigCuenta;
  final String rutaSteamCloudCuenta;

  String get nombreVisible =>
      nombre == steamId ? steamId : '$steamId - $nombre';

  @override
  String toString() => nombreVisible;
}

File: project/D2Sync/Models/CuentaSteam.cs
namespace D2Sync.Models;

public class CuentaSteam
{
    public required string SteamId { get; init; }
    public required string Nombre { get; init; }
    public required string RutaCuenta { get; init; }
    public required string RutaConfigCuenta { get; init; }
    public required string RutaSteamCloudCuenta { get; init; }

    public string NombreVisible => Nombre == SteamId ? SteamId : $"{SteamId} - {Nombre}";

    public override string ToString() => NombreVisible;
}
